CMakeLists.txt

```diff
@@ -549,12 +549,19 @@ set_target_properties(${TARGET} PROPERTIES PUBLIC_HEADER "ggml.h;whisper.h")
 
 include(GNUInstallDirs)
 
+set(WHISPER_INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_INCLUDEDIR}
+    CACHE PATH "Location of header files")
+set(WHISPER_LIB_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR}
+    CACHE PATH "Location of library files")
+set(WHISPER_BIN_INSTALL_DIR ${CMAKE_INSTALL_BINDIR}
+    CACHE PATH "Location of binary files")
+
 install(TARGETS ${TARGET}
-    LIBRARY  DESTINATION lib
-    ARCHIVE  DESTINATION lib/static
-    RUNTIME  DESTINATION bin
-    RESOURCE DESTINATION bin
-    PUBLIC_HEADER DESTINATION include
+    LIBRARY  DESTINATION "${WHISPER_LIB_INSTALL_DIR}"
+    ARCHIVE  DESTINATION "${WHISPER_LIB_INSTALL_DIR}/static"
+    RUNTIME  DESTINATION "${WHISPER_BIN_INSTALL_DIR}"
+    RESOURCE DESTINATION "${WHISPER_BIN_INSTALL_DIR}"
+    PUBLIC_HEADER DESTINATION "${WHISPER_INCLUDE_INSTALL_DIR}"
```

Comment:
```suggestion
```
If the custom directories are removed, you can use the defaults, which are already compatible with windows (at least through vcpkg)